src/test/ui/suggestions/suggest-methods.stderr

```diff
@@ -5,19 +5,19 @@ LL | struct Foo;
    | ----------- method `bat` not found for this
 ...
 LL |     f.bat(1.0);
-   |       ^^^ help: there is a method with a similar name: `bar`
+   |       ^^^ help: there is a associated function with a similar name: `bar`
```

Comment:
```suggestion
   |       ^^^ help: there is an associated function with a similar name: `bar`
```

Reply:
NOTE TO SELF: I will need to fix the article...